eiger.cscs.ch/multicore-mr0/computer-configure.yaml

```
@@ -1,3 +1,15 @@
 ---
-safe_interval: 60
-proxy_command: ssh -q -Y {username}@ela.cscs.ch netcat eiger.cscs.ch 22
+port: 22
+look_for_keys: true
+timeout: 60
+allow_agent: true
+proxy_jump: ''
```

It only works for me when adding this.
```suggestion
proxy_jump: 'ela.cscs.ch'
```